docs/pages/brownfield/installing-expo-modules.mdx

```diff
@@ -0,0 +1,193 @@
+
+## Prerequisites
+
+You should have a native project with React Native installed and configured to render a root view. If you don't have this yet, you can follow [the "Integration with Existing Apps" guide](https://reactnative.dev/docs/integration-with-existing-apps) on the React Native docs and then return here.
+
+## Install the `expo` package
+
+First, add the `expo` package to your project. Ensure you are using a version of the `expo` package that is compatible with the React Native version in your project. [Learn more](/versions/latest/#each-expo-sdk-version-depends-on-a-react-native-version).
+
+<Terminal cmd={['npm install expo']} />
+
+## Configure your iOS app
+
+<Step label="1">
+  Add the following to your `Podfile` in the `ios` directory:
+  <DiffBlock
+    raw={`
+diff --git a/ios/Podfile b/ios/Podfile
+index 2d3a979..825a9d4 100644
+--- a/ios/Podfile
++++ b/ios/Podfile
+@@ -1,3 +1,4 @@
++require File.join(File.dirname(\`node --print "require.resolve('expo/package.json')"\`), "scripts/autolinking")
+ require Pod::Executable.execute_command('node', ['-p',
+   'require.resolve(
+   "react-native/scripts/react_native_pods.rb",
+@@ -14,6 +15,7 @@ if linkage != nil
+ end
+ 
+ target '<YourAppTarget>' do
++  use_expo_modules!
+   config = use_native_modules!
+  `}
+  />
+</Step>
+
+<Step label="2">
+  Open your `ios` directory in XCode. From the project navigator, select your project and then
+  select your app target under `TARGETS`. In `Build Settings`, using the search bar, search for
+  `ENABLE_USER_SCRIPT_SANDBOXING`. If it is not already, set its value to `No`.
+</Step>
+
+<Step label="3">
+  Run `pod install` in the `ios` directory.
+  <Terminal cmd={['cd ios && pod install']} />
+  You will need to do this everytime you add a dependency that uses native code.
+</Step>
+
+<Step label="4">
+  **(Optional)** Complete the following if you would like to use [AppDelegate subscribers](/modules/appdelegate-subscribers/).
+  <DiffBlock
+    raw={`diff --git a/ios/<MyAppProject>/AppDelegate.swift b/ios/<MyAppProject>/AppDelegate.swift
+index ff83531..bd8651d 100644
+--- a/ios/<MyAppProject>/AppDelegate.swift
++++ b/ios/<MyAppProject>/AppDelegate.swift
+@@ -1,31 +1,29 @@
+ import UIKit
++import ExpoModulesCore
+ 
+ @main
+-class AppDelegate: UIResponder, UIApplicationDelegate {
++class AppDelegate: ExpoAppDelegate {
+ 
+-  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
++  override func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+-    return true
++    super.application(application, didFinishLaunchingWithOptions: launchOptions)
+   }
+ `}
+  />
+</Step>
+
+## Configure your Android app
```

Comment:
Suggestion: move this whole section above "Configure your iOS app" to match the (Android, iOS) ordering we use elsewhere mostly.